search/src/main/scala/weco/api/search/rest/ImagesParams.scala

```diff
@@ -42,6 +42,7 @@ case class MultipleImagesParams(
   license: Option[LicenseFilter],
   `source.contributors.agent.label`: Option[ContributorsFilter],
   `source.genres.label`: Option[GenreFilter],
+  `source.genres.concepts.id`: Option[GenreConceptFilter],
```

Comment:
When the final part of a filter is the `.id`, we omit the `.id` – this filter should be called `source.genres.concepts`, and similarly for `genre.concepts` in the catalogue API.

See https://github.com/wellcomecollection/docs/tree/main/rfcs/037-api-faceting-principles#principles, which we should probably put somewhere more prominent.

Reply:
Good point. Now you mention it, I remember this from ages ago.